packages/jumpstarter/jumpstarter/common/utils.py

```
@@ -76,6 +75,11 @@ def launch_shell(
         if cmd[0].endswith("bash"):
             cmd.append("--norc")
             cmd.append("--noprofile")
+            env["PS1"] = f"{ANSI_GRAY}{PROMPT_CWD} {ANSI_YELLOW}⚡{ANSI_WHITE}{context} {ANSI_YELLOW}➤{ANSI_RESET} "
```

Pylint (3.3.7)

[convention] Line too long (115/100)

(C0301)